enclave-protocol/src/tdbe_protocol.rs

```diff
@@ -0,0 +1,35 @@
+use std::borrow::Cow;
+
+use parity_scale_codec::{Decode, Encode};
+
+use chain_core::tx::{data::TxId, TxWithOutputs};
+
```

Comment:
may be good to create the response data type for enclave->untrusted/runner
where it'll return the *sealed* transactions or the node's keypackage

Reply:
Split the protocol in trusted/untrusted parts